FSharpLu.Tests/JsonTests.fs

```diff
@@ -252,6 +268,20 @@ type JsonSerializerTests() =
         areReciprocal <| (Option.Some <| SomeAmbiguity.DUWithCaseWithFieldNamedSome.Some)
 
     [<TestMethod>]
+    [<TestCategory("FSharpLu.Json.CamelCase")>]
+    member __.``CamelCaseSerializer handles discriminated unions`` () =
+        let du = ComplexDu <| SomeField (2, 3)
+        let str = CamelCaseSerializer.serialize du
+        Assert.AreEqual("""{"complexDu":{"someField":[2,3]}}""", str)
+
+    [<TestMethod>]
```

Comment:
I would add some FsCheck tests similar to `No ambiguity between records and Option type` and `Serialization and deserialization are reciprocal` to make sure that the case insensitive string comparison does not brake anything when `Some` types are nested.